Add documentation about `SeccompDefault` feature (#27957)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates.md

| `RotateKubeletServerCertificate` | `false` | Alpha | 1.7 | 1.11 |
| `RotateKubeletServerCertificate` | `true` | Beta | 1.12 | |
| `RunAsGroup` | `true` | Beta | 1.14 | |

> | `SeccompDefault` | `false` | Alpha | 1.22 | 1.23 |

Reviewer: Because v1.23 is a future version (?), you can leave the last column empty.

Author: Yep, done :+1:

  instead of the DaemonSet controller.
- `SCTPSupport`: Enables the _SCTP_ `protocol` value in Pod, Service,
  Endpoints, EndpointSlice, and NetworkPolicy definitions.

> - `SeccompDefault`: Enables using the `RuntimeDefault` seccomp profile if nothing

Reviewer: nit: Could you provide more detail? What is:
"if nothing else is specified within the ..."

Could it be: If the `SecurityContext` is not set?

Reviewer 2: Or potentially:

```suggestion
- `SeccompDefault`: Enables the use of `RuntimeDefault` as the default seccomp profile. 
```
